client/package.json

```diff
@@ -41,6 +43,6 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }, 
+  },
   "proxy": "http://localhost:5000"
-}
+}
```

Comment:
Hello Khamis, you need to add line here :) 
I think you can fix this by changing the default formater in .vscode
Please add this to .vscode setting 
 "editor.defaultFormatter": "dbaeumer.vscode-eslint",

Reply:
Thx Hasan :heart: